src/player_records.rs

```diff
@@ -156,24 +157,19 @@ impl DerefMut for PlayerRecords {
 // PlayerRecord
 
 /// A Record of a player stored in the persistent personal playerlist
-#[derive(Debug, Serialize, Deserialize, Clone)]
+#[derive(Debug, Serialize, Deserialize, Clone, Default)]
 pub struct PlayerRecord {
     #[serde(default = "default_custom_data")]
     pub custom_data: serde_json::Value,
     pub verdict: Verdict,
-    #[serde(default)]
     pub previous_names: Vec<Arc<str>>,
+    #[serde(default = "default_date")]
+    pub date_modified: DateTime<Utc>, // updates on any manual change by user.
```

Comment:
You can put a comment above a field with triple slash and it will be used as a docstring. e.g.
```rust
/// Updates on any manual change made by the user
#[serde(...)]
pub date_modified: DateTime<Utc>,
```

Reply:
Very inconsequential, but just letting you know.